src/placeos-rest-api/controllers/drivers.cr

```diff
@@ -97,9 +97,10 @@ module PlaceOS::Api
       # Set the repository commit hash to head
       driver.update_fields(commit: "RECOMPILE-#{driver.commit}")
 
-      # Wait until the commit hash is not head with a timeout of 20 seconds
-      Utils::Changefeeds.await_model_change(driver, timeout: 20.seconds) do |update|
-        update.destroyed? || !update.commit.starts_with? "RECOMPILE"
+      # Wait until the commit hash is not head with a timeout of 90 seconds
+      # ameba:disable Style/RedundantReturn
+      return Utils::Changefeeds.await_model_change(driver, timeout: 90.seconds) do |update|
```

Comment:
What's the `return` doing?

Reply:
It doesn't work without the return I tried

Reply:
Even though it should...